scripts/integer-tests.sh

```diff
@@ -24,35 +24,64 @@ else
     n_threads=6
 fi
 
-# block pbs are too slow for high params
-# mul_crt_4_4 is extremely flaky (~80% failure)
-# test_wopbs_bivariate_crt_wopbs_param_message generate tables that are too big at the moment
-# same for test_wopbs_bivariate_radix_wopbs_param_message_4_carry_4
-# test_integer_smart_mul_param_message_4_carry_4 is too slow
-filter_expression=''\
+if [[ "${BIG_TESTS_INSTANCE}" == FALSE ]]; then
+    # block pbs are too slow for high params
```

Comment:
The filter expression block seems to be duplicated in both conditional statements. Can't we get it out of the if statement and declare it once ?

Reply:
they are not exactly the same, the BIT_TESTS_INSTANCE == FALSE case runs the 4_4 tests separately while the TRUE case just runs everything at once